sdk/test-utils/test-utils/test/tracing/azureTracing.ts

```diff
@@ -0,0 +1,41 @@
+import { useInstrumenter } from "@azure/core-tracing";
+import { assert } from "chai";
+import { TestInstrumenter } from "../../src";
+
+// this is the plugin used in the test file
+function chaiAzureTrace(chai: Chai.ChaiStatic, _utils: Chai.ChaiUtils) {
+  // expect(() => {}).to.supportsTracing() syntax
+  chai.Assertion.addMethod("supportsTracing", function() {
+    this._obj.call(undefined);
+  });
+  // assert.supportsTracing(() => {}) syntax
+  chai.assert.supportsTracing = supportsTracing;
+}
+
+// Implementation
+async function supportsTracing<Callback extends (...args: unknown[]) => unknown>(
```

Comment:
So right now we have:

```ts
expect(() => {}).to.supportsTracing()
assert.supportsTracing(() => {})
```
Maybe we only need to support assert style? (that's the line 12 one)

Reply:
S with our latest implementation we are supporting both the syntax